.eslintrc.js

```diff
@@ -21,5 +22,18 @@ module.exports = {
     __VERSION__: true,
     CSS: true
   },
-  plugins: ['flowtype']
+  overrides: [
```

Comment:
don't we have the same problem with other repos? we could add this to https://github.com/cssinjs/eslint-config-jss

Reply:
cssinjs/eslint-config-jss#4